lib/glific/contacts.ex

```diff
@@ -644,7 +644,7 @@ defmodule Glific.Contacts do
   @spec optin_contact(map()) ::
           {:ok, Contact.t()} | {:error, Ecto.Changeset.t()} | {:error, String.t()}
   def optin_contact(%{organization_id: organization_id} = attrs) do
-    organization = Partners.organization(organization_id)
```

Comment:
@AkhileshNegi what was the need to changing this function?
I think in the organization function we are managing all the cache and also adding bsp and other services.

Reply:
@pankaj-ag  Reverted changes. I used different input in API and _Partners.organization_ was giving error. I reverted it now as I also removed the earlier changes